pkgs/build-support/fetchzip/default.nix

```diff
@@ -5,16 +5,20 @@
 # (e.g. due to minor changes in the compression algorithm, or changes
 # in timestamps).
 
-{ fetchurl, unzip }:
+{ lib, fetchurl, unzip }:
 
 { # Optionally move the contents of the unpacked tree up one level.
   stripRoot ? true
-, url
+, url ? ""
+, urls ? []
 , extraPostFetch ? ""
 , name ? "source"
 , ... } @ args:
 
-(fetchurl ({
+(fetchurl (let
+  # TODO: don't assume all basenames in urls are the same
```

Comment:
I am not sure what is the actionable art of that TODO. 

I do not think it is a good idea to assert that; I think just considering the first URL to be the «main» one and using its basename is perfectly acceptable, no?

Reply:
After taking a second look this should be fine because we only use the name to copy files to, not from.